dev/com.ibm.ws.threading/src/com/ibm/wsspi/threading/package-info.java

```diff
@@ -1,17 +1,17 @@
 /*******************************************************************************
- * Copyright (c) 2010 IBM Corporation and others.
+ * Copyright (c) 2025 IBM Corporation and others.
```

Comment:
Need to leave original copyright year in file unless it was incorrect
```suggestion
 * Copyright (c) 2010,2025 IBM Corporation and others.
```

Reply:
latest commit resolves this